cpp/benchmarks/hausdorff_benchmark.cpp

```diff
@@ -51,6 +49,11 @@ static void BM_hausdorff(benchmark::State& state)
 }
 
 class HausdorffBenchmark : public cuspatial::benchmark {
+  virtual void SetUp(const ::benchmark::State& state) override
+  {
+    mr = std::make_shared<rmm::mr::cuda_memory_resource>();
+    rmm::mr::set_current_device_resource(mr.get());  // set default resource to cuda
+  }
```

Comment:
Why is this needed? The default resource *is* the `cuda_memory_resource`.

Reply:
The benchmark base class instantiates an RMM pool resource in the `SetUp` function. It's assigned to a [local variable](https://github.com/rapidsai/cuspatial/blob/branch-0.19/cpp/benchmarks/fixture/benchmark_fixture.hpp#L70) instead of the member var, so it segfaults. When I change fix that, I get "Maximum pool size exceeded," on the first bench run.

I'm not sure why the second error is happening, and this PR is primarily about CMake. So this change just makes the benchmark code compile and run without crashing.